src/Providers/DataGrid/Wijmo/Features/ToolTip.ts

```diff
@@ -37,7 +38,14 @@ namespace Providers.DataGrid.Wijmo.Feature {
                     );
                 }
             } else if (cellType === wijmo.grid.CellType.ColumnHeader) {
-                this._setHeaderTooltip(_currTarget, ht.col);
+                // If the Column Header is from a Group Column, we need to use a different approach that the regular header
+                if (
+                    _currTarget.classList.contains(
+                        Helper.Constants.CssClasses.ColumnGroup
+                    )
+                )
+                    this._setColumnGroupHeaderTooltip(_currTarget);
+                else this._setHeaderTooltip(_currTarget, ht);
```

Comment:
Even if it's one line only, please use `if (...) { ... } else { ... }`.

Reply:
Done!